src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```
@@ -9,6 +9,7 @@ parameters:
 
   # The following parameters aren't expected to be passed in rather they are used for encapsulation
   # -----------------------------------------------------------------------------------------------
+  alpine317Container: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-20230214152931-7f13c75
```

We want this to be latest. Following the pattern of the other image names is not what we want. I'll update those separately.

```suggestion
  alpine317Container: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17
```